.gitignore

```diff
@@ -1,4 +1 @@
-src/Resources/app/storefront/dist
 node_modules
-src/Resources/app/storefront/src/js/glightbox.js
-src/Resources/app/storefront/src/scss/vendors/glightbox.scss
```

Comment:
Those files were excluded from git for a reason.
During every theme:install process a new version of GLightbox (if compatible with 2.0.5) is downloaded and put in those directories. So they would be added to the commit. The point was not to trash commits and no to remove them manually every time you make a commit.
I would leave them in .gitignore

Reply:
As you can see in https://github.com/notebooksbilliger/ShopWare6-MainTheme/pull/17/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R13, I've modified the script slightly to create `*.scss` files directly in `node_modules` dir which is actually gitignored to avoid polluting `src/Resources/app/storefront/src`.
SCSS vendor files can be imported directly from node_modules: https://github.com/notebooksbilliger/ShopWare6-MainTheme/pull/17/files#diff-5408e260abe82780490f34de727ae9dcR11

Js files don't need to be copied at all - they can be imported in `main.js` entrypoint either with `import` or `require` directive eg. https://github.com/notebooksbilliger/ShopWare6-MainTheme/pull/17/files#diff-de91c84e345bb53ccd2813aa51a50de0R7